extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -741,6 +741,10 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
     return {
       PAGEVIEWID: () => Services.documentInfoForDoc(this.element).pageViewId,
       HREF: () => this.win.location.href,
+      TGT: () => {
+        const json = tryParseJson(this.element.getAttribute('data-json'));
```

Comment:
TGT: () => (tryParseJson(this.element.getAttribute('data-json')) || {})['targeting'] || '',

Reply:
![image](https://user-images.githubusercontent.com/4276992/34725771-f461e6b0-f51f-11e7-918e-32275cd0b43d.png)